src/elexmodel/client.py

```diff
@@ -56,21 +56,27 @@ def _check_input_parameters(
         offices = config_handler.get_offices()
         if office not in offices:
             raise ValueError(f"Office '{office}' is not valid. Please check config.")
-        valid_estimands = config_handler.get_estimands(office)
-        for estimand in estimands:
-            if estimand not in valid_estimands:
-                raise ValueError(f"Estimand: '{estimand}' is not valid. Please check config")
+
+        estimands_in_config = config_handler.get_estimands(office)
+        extra_estimands = set(estimands).difference(set(estimands_in_config))
+        if len(extra_estimands) > 0:
+            # this is ok; later they'll all be created by the Estimandizer
+            LOG.info("Found additional estimands that were not specified in the config file: %s", extra_estimands)
```

Comment:
if the user didn't create a new estimand function but just input something random, then this would pass, right?

Reply:
Yes and no.  Eventually you'll get a `KeyError` when the "something_random" estimand's function can't be found.  I figured a warning here would be helpful for both (a) you have an estimand that you didn't specify in your config and (b) if that `KeyError` does happen, here's an indication as to why 😄

Reply:
that sounds good, thanks!